administrator/components/com_admin/sql/updates/mysql/4.1.5-2022-06-02.sql

```diff
@@ -0,0 +1,32 @@
+UPDATE `#__extensions`
+   SET `params` = REPLACE(`params`, '"negotiate_tls":1', '"encryption":"tls"')
+ WHERE `name` = 'plg_authentication_ldap'
+   AND `type` = 'plugin'
```

Comment:
```suggestion
   AND `type` = 'plugin'
   AND `element` = 'ldap'
```
Sorry, the wrong element was my fault. But it should be corrected, not removed.

Reply:
No problem, I assumed I had checked that and now had a different installation. I applied the fix, thanks for reviewing.